recipes/ctng-compilers/meta.yaml

```
@@ -0,0 +1,258 @@
+{% set build_num = ctng_gcc_build_number %}
+
+package:
+  name: ctng-compilers
+  version: {{ ctng_gcc }}
+
+source:
+  path: .
+
+build:
+  merge_build_host: False
+  # defined in conda_build_config.yaml so that we can keep it associated with a particular GCC version number
+  number: {{ build_num | default(0) }}
+  skip: True  # [not linux]
+  # The CentOS6 system libraries should be used for everything here.
+  # We do not have CDT packages yet, nor compilers that would use them!
+  missing_dso_whitelist:
+    - "*"
+
+requirements:
+  build:
+    - crosstool-ng >=1.23.0
+    - bash  # [osx]
```

Should the build dependencies be expanded?
```suggestion
    - bash  # [osx]
    - make
    - findutils
    - rsync
```
(it should probably be in `outputs/requirements/build` instead of the top level one)